contracts/src/v0.8/automation/dev/test/AutomationRegistry2_3.t.sol

```diff
@@ -504,3 +505,100 @@ contract AutomationRegistry2_3_SetConfig is AutomationRegistry2_3_SetUp {
     return bytes32((prefix & prefixMask) | (h & ~prefixMask));
   }
 }
+
+contract AutomationRegistry2_3_NOPsSettlement is AutomationRegistry2_3_SetUp {
+  event NOPsSettledOffchain(address[] transmitterList, uint256[] balances);
```

Comment:
Not saying we have to do this here... but I really like [this guys's design](https://betterprogramming.pub/a-solidity-symphony-testing-solidity-event-emissions-with-foundry-5e241415796b) for separating the events into a separate interface so that they can easily be used in tests without having to redefine the event.

Reply:
this is a good idea. we should do this in registry at some point. maybe 3.0!